### ISSUE TITLE
datetime support

### ISSUE THREAD

**Issue by sashaostr** [julianpeeters/avrohugger#41]
Hi,
Do you have any intent to support logical types of time-millis etc.?
If not, where would be the right place inside your code to add this support?
Thanks!

**Comment by julianpeeters** [comment]
Hi Alexander,

I have no plans to add support myself in the near future, but I agree that they would be nice to have and I'd be happy to merge a PR if you're considering it.

I haven't really thought fully through how logical types would be supported. At one time there was a thought to [add decimals via logical types](https://github.com/julianpeeters/avrohugger/issues/29), and I don't really have much more advice to add to what I wrote there, but I'm happy to answer any questions if you decide to try to add this feature.

**Comment by mariussoutier** [comment]
I think the biggest downside right now is that logical types are not available in AVDLs, only in AVSCs.

**Comment by francescopellegrini** [comment]
Does 1e0a1e3 change any of these plans?

**Comment by julianpeeters** [comment]
@francescopellegrini yes, it appears that logical types are being slowly supported in avdl, the tentative plan with avrohugger is to start with decimals. 
https://avro.apache.org/docs/1.8.2/idl.html
```
Logical Types
Some of the logical types supported by Avro's JSON format are also supported by Avro IDL. The currently supported types are:

decimal (logical type decimal)
date (logical type date)
time_ms (logical type time-millis)
timestamp_ms (logical type timestamp-millis)
For example:

record Job {
  string jobid;
  date submitDate;
  time_ms submitTime;
  timestamp_ms finishTime;
  decimal(9,2) finishRatio;
}
```

**Comment by julianpeeters** [comment]
@francescopellegrini fyi date and timestamp are also planned (in addition to decimal), but initial support will be limited to `Standard` format for now (PRs welcome for `SpecificRecord`)

**Comment by julianpeeters** [comment]
fixed in avrohugger 1.0.0-RC10 and sbt-avrohugger 2.0.0-RC10
